Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/IntakeEject.java

```diff
@@ -17,5 +21,46 @@ public void initDefaultCommand() {
         // Set the default command for a subsystem here.
         //setDefaultCommand(new MySpecialCommand());
     }
+
+    /**
+	 * returns current motor value
+	 */
+	public double getIntake() {
+		return intakeMotors.get();
+	}
+	
+	/**
+	 * Uses (insert sensor here) to detect 
+	 * a cube in front of the robot.
+	 */
+	public boolean detectCube() {
+		return false;
+	}
+	
```

Comment:
I might change the method name detectCube() to something like seesCube() to make the difference between this method and the one below (hasCube) a little clearer. In the end this method is something that might become obsolete because the driver will just look through the camera and see it, but good to have here in case we are able to automate enough :)